docs/projectSetup/android/checklist_android.md

```diff
@@ -0,0 +1,44 @@
+# Android Project Setup
+## Git
+1. Create new repository based on [native-android-repo-template](https://github.com/nanogiants/native-android-repo-template)
+
+## Android Studio
+1. Create an empty base project in Android Studio.
+    1. Configure build types for different environments.
+        - TODO: List all build types (develop, staging, production) and add basic gradle snippets to template repo
+    2. Create a signing keystore for release builds.
```

Comment:
What is best practise here? Do we have separate keystores per environment (e.g. staging, production) or is it safe to share the keystore in multiple environments (except production obviously)